### PR TITLE
Code formatting 

### PR DESCRIPTION
**Pull request by tejash-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
Code formatting 
When this pull request is merged, it will...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=2457) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=2457) before we can accept your contribution.<br/>

**Comment by mhauri** [comment]
Hi @tejash-wagento thank you for your contribution.
Would you mind to merge all your changes from the other Pull Requests you opened into this one and to close all others?

#2466
#2464 
#2463
#2462 
#2461 
#2460 
#2459
#2458

**Comment by tejash-wagento** [comment]
@mhauri 
I have merged my all branches in patch-1.

**Comment by jeff-matthews** [comment]
We also need to change the base branch to `master` @mhauri. You probably can't do that without write access to the repo, so I'll change it.

File: guides/m1x/install/installer-privileges_after.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h2 id="terms">Terminology</h2>
     <h2 id="privs-after">Setting Privileges and Ownership After You Install Magento</h2>
     <p>If you have installed Magento, you can set file system privileges and ownership as follows:</p>
     <ul><li>For a dedicated Magento server, you set ownership of files and directory as the web server user. You set privileges as 500 (directories) and 400 (files).</li>
-        <li>For a hosted Magento server on which the web server runs as the logged-in username, you set privileges as as 500 (directories) and 400 (files).</li></ul>
+        <li>For a hosted Magento server on which the web server runs as the logged-in username, you set privileges as 500 (directories) and 400 (files).</li></ul>
```
Comment (sanganinamrata) [inline guides/m1x/install/installer-privileges_after.html]: @tejash-wagento, Hi
These changes are already fixed and accepted. in this PR: https://github.com/magento/magento2/pull/16644

File: guides/v2.0/config-guide/varnish/config-varnish-configure.md [hunk 2]
```diff
@@ -199,7 +199,7 @@ A long list of response headers display in your command prompt window. Look for
 	-   VCL_call       BACKEND_FETCH
 	-   VCL_return     fetch
 	-   BackendOpen    17 default(10.249.151.10,,8080) 10.249.151.10 60914
-	-   Backend        17 default default(10.249.151.10,,8080)
+	-   Backend        17 default(10.249.151.10,,8080)
```
Comment (sanganinamrata) [inline guides/v2.0/config-guide/varnish/config-varnish-configure.md]: @tejash-wagento, Hi
These changes are already fixed and accepted. in this PR: https://github.com/magento/devdocs/pull/2448

File: guides/v2.1/config-guide/varnish/config-varnish-configure.md [hunk 3]
```diff
@@ -199,7 +199,7 @@ A long list of response headers display in your command prompt window. Look for
 	-   VCL_call       BACKEND_FETCH
 	-   VCL_return     fetch
 	-   BackendOpen    17 default(10.249.151.10,,8080) 10.249.151.10 60914
-	-   Backend        17 default default(10.249.151.10,,8080)
+	-   Backend        17 default(10.249.151.10,,8080)
```
Comment (sanganinamrata) [inline guides/v2.1/config-guide/varnish/config-varnish-configure.md]: @tejash-wagento, Hi
These changes are already fixed and accepted. in this PR: https://github.com/magento/devdocs/pull/2448

**Comment by jeff-matthews** [comment]
@tejash-wagento,

We haven't heard from you regarding this PR in two weeks, so we're closing this PR due to inactivity. If you have further comments and feedback, please create a new PR. Thanks for taking the time to create a GitHub issue for docs improvement!

**Pull request closed without merging**